docs/byoh-instance-pre-requisites.md

```diff
@@ -2,8 +2,9 @@
 
 The following pre-requisites must be fulfilled in order to add a Windows BYOH node.
 * The instance must be on the same network as the Linux worker nodes in the cluster.
-* Port 22 must be open and running [an SSH server](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse).
-* Port 10250 must be open in order for log collection to function.
+* Port 22 must allow inbound TCP traffic and be running [an SSH server](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse).
```

Comment:
It may be out of scope, but should we specify that we only recommend OpenSSH since that's the only SSH server we test with?

Reply:
Thats out of scope, I'm just changing the language of that line to be consistent with the others lines.

But to answer your question, yes I think we could state that